artiq/coredevice/coredevice_generic.schema.json

```diff
@@ -168,6 +168,10 @@
                         "bank_direction_high": {
                             "type": "string",
                             "enum": ["input", "output"]
+                        },
+                        "custom_ififo_lengths": {
+                            "type": "array",
```

Comment:
Why not a dictionary?

Reply:
Is there a dictionary object in json?

Reply:
Sure

Reply:
If I use the `{"0": 1024}` syntax, I think the channel number has to be a string (while it should be an int). If you think that is better, I can make that change.

Reply:
That's standard enough that ``json.dumps`` silently converts the keys to strings.